src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaFixture.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.E2ETest;
+
+namespace Microsoft.Spark.Extensions.Delta.E2ETest
+{
+    public class DeltaFixture
+    {
+        public SparkFixture SparkFixture { get; private set; }
+
+        public DeltaFixture()
+        {
+            // Set environment variables.
+            Environment.SetEnvironmentVariable(
+                SparkFixture.EnvironmentVariableNames.Packages,
+                "io.delta:delta-core_2.11:0.3.0");
+
+            SparkFixture = new SparkFixture();
```

Comment:
If you are doing this can't you have overloaded constructor to take in the setting instead of setting it via env variable?

Reply:
Fixture can't have an overloaded constructor.  I tried this and xUnit throws errors.

Reply:
Fixture doesn't support overloaded constructor.  I tried it and xUnit throws exceptions.

Reply:
OK. in that case, this approach is fine.